pkg/client/container.go

```diff
@@ -71,7 +93,7 @@ func (c Client) GetContainer(ctx context.Context, id *container.ID, opts ...Call
 // which checks if the structure of the resulting container matches its identifier.
 //
 // Returns container.ErrIDMismatch if container does not match the identifier.
-func GetVerifiedContainerStructure(ctx context.Context, c *Client, id *container.ID, opts ...CallOption) (*container.Container, error) {
+func GetVerifiedContainerStructure(ctx context.Context, c *clientImpl, id *container.ID, opts ...CallOption) (*container.Container, error) {
```

Comment:
Lets
```suggestion
func GetVerifiedContainerStructure(ctx context.Context, c Client, id *container.ID, opts ...CallOption) (*container.Container, error) {
```
? Same for `func DeleteObject`

Reply:
Yes, missed that.